mongodbatlas/mongodbatlas.go

```diff
@@ -564,3 +566,38 @@ func setListOptions(s string, opt interface{}) (string, error) {
 	origURL.RawQuery = origValues.Encode()
 	return origURL.String(), nil
 }
+
+// ServiceVersion represents version information.
+type ServiceVersion struct {
+	GitHash string
+	Version string
+}
+
+// String serializes VersionInfo into string.
+func (v *ServiceVersion) String() string {
+	return fmt.Sprintf("gitHash=%s; versionString=%s", v.GitHash, v.Version)
+}
+
+func parseVersionInfo(s string) *ServiceVersion {
+	if s == "" {
+		return nil
+	}
+
+	var result ServiceVersion
+	pairs := strings.Split(s, ";")
+	for _, pair := range pairs {
+		keyvalue := strings.Split(strings.TrimSpace(pair), "=")
+		switch keyvalue[0] {
+		case "gitHash":
+			result.GitHash = keyvalue[1]
+		case "versionString":
+			result.Version = keyvalue[1]
+		}
```

Comment:
[q] Should we log/fail on any other case? I generally don't like switches without a default, as changes can go undetected. If we had a default error, along with a unit test, a structure change would immediately fail here.